source/release-notes/v1.8-release-notes.rst

```diff
@@ -102,38 +103,149 @@ The Open OnDemand Portal :ref:`ood-portal-generator-configuration` file now supp
 Beta CCQ Scheduler support
 ..........................
 
+We now have beta `Cloudy Cluster`_ support for their ccq scheduler.  You can find
+:ref:`documentation on how to configure the cluster here <resource-manager-cloudy-cluster>`.
+
 Retain completed jobs panel for debugging
 .........................................
 
+When an interactive job completes, its panel will now remain in the view. This is to
+allow users and administrators to debug failed jobs by keeping the session id links
+in view.
+
 Batch Connect apps now support multiple clusters
 ................................................
 
+Batch connect applications now support multiple clusters in their configuration
+instead of just one.
+
+Where previous configuration look like this, with only a single cluster.
+
+.. code-block:: yaml
+
+   cluster: "my_cluster"
+
+Configurations will now accept an array like the example below.  This will create a
+dropdown select menu item at the top of the form where users can choose between
+the two options.
+
+.. code-block:: yaml
+
+   cluster:
+     - "my_cluster"
+     - "my_other_cluster"
+
+
+There are also other ways to configure which cluster to submit to.
+:ref:`Complete documentation can be found here. <configuring-cluster>`
+
 Batch Connect apps now support disabling caching on fields
 ..........................................................
 
+Batch connect apps can now enable or disable caching site wide, on a per app basis
+or on a per item basis.
```

Comment:
We need a little more explanation so users know what we are talking about when we say "caching in fields". Something like:

>When a user submits the web form for an interactive app (batch connect) the form values the user set are stored in a cache file that is used to pre-populate the web form in the future. In some cases, setting the default value of the form field to a previously used value is undesirable. You can now disable this caching feature for every app, on a per app basis, or on a per field basis.